backend/spiritual_avatar_generation_engine.py

```
@@ -21,6 +21,8 @@
 
 from core_foundation_enhanced import settings, db_manager
 from enhanced_business_logic import SpiritualAvatarEngine, AvatarGenerationContext, AvatarEmotion
+from ..schemas.avatar import AvatarSessionCreate, AvatarSession
```

Ruff (0.12.2)

`..schemas.avatar.AvatarSessionCreate` imported but unused

Remove unused import: `..schemas.avatar.AvatarSessionCreate`

(F401)